helm-charts/whitesource-renovate/templates/deployment.yaml

```
@@ -0,0 +1,155 @@
+                  key: renovateToken
+            {{- end }}
+            {{- if .Values.renovate.githubAppId }}
+            - name: GITHUB_APP_ID
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Release.Name }}
+                  key: githubAppId
+            {{- end }}
+            {{- if .Values.renovate.githubAppKey }}
+            - name: GITHUB_APP_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Release.Name }}
+                  key: githubAppKey
+            {{- end }}
+            {{- if .Values.renovate.githubComToken }}
+            - name: GITHUB_COM_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Release.Name }}
+                  key: githubComToken
+            {{- end }}
+            {{- if .Values.renovate.webhookSecret }}
+            - name: WEBHOOK_SECRET
+              value: {{ .Values.renovate.webhookSecret | quote }}
+            {{- end }}
+            {{- if .Values.renovate.schedulerCron }}
+            - name: SCHEDULER_CRON
+              value: {{ .Values.renovate.schedulerCron | quote }}
+            {{- end }}
+            {{- if .Values.renovate.pipIndexUrl }}
+            - name: PIP_INDEX_URL
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Release.Name }}
+                  key: pipIndexUrl
+            {{- end }}
+            {{- if .Values.renovate.noNodeTlsVerify }}
+            - name: NODE_TLS_REJECT_UNAUTHORIZED
+              value: '0'
+            {{- end }}
+            {{- if .Values.renovate.noGitTlsVerify }}
+            - name: GIT_SSL_NO_VERIFY
+              value: 'true'
+            {{- end }}
+            {{- if .Values.renovate.renovateUserAgent }}
+            - name: RENOVATE_USER_AGENT
+              value: {{ .Values.renovate.renovateUserAgent | quote }}
+            {{- end }}
+            {{- if .Values.renovate.logLevel }}
+            - name: LOG_LEVEL
+              value: {{ .Values.renovate.logLevel | quote }}
+            {{- end }}
+          ports:
+            - name: http
+              containerPort: 8080
+              protocol: TCP
+          livenessProbe:
+            initialDelaySeconds: 60
+            httpGet:
+              path: /
+              port: http
+          readinessProbe:
+            httpGet:
+              path: /
+              port: http
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
+          volumeMounts:
+            - name: {{ .Release.Name }}-config-js-volume
+              mountPath: /usr/src/app/config.js
+              subPath: config.js
+            {{- if .Values.renovate.npmrc }}
+            - name: {{ .Release.Name }}-npmrc-volume
+              mountPath: /home/ubuntu/.npmrc
+              subPath: .npmrc
+            {{- end }}
+            - name: {{ .Release.Name }}-cache-volume
+              mountPath: /tmp
```

```suggestion
              mountPath: /tmp/renovate
```

In an earlier version this already was `/tmp/renovate`. I think this made sense because putting `/tmp` on the volume would also put system temp files on it.

---

Ok for me but Docker compose files will have to be updated too